control/trajectory_follower/include/trajectory_follower/sync_data.hpp

```diff
@@ -31,6 +31,7 @@ struct LateralSyncData
 struct LongitudinalSyncData
 {
   bool is_velocity_converged{false};
+  int state_of_longitudinal_controller{2};
```

Comment:
It's ok for draft, but we want change to use
```
enum class ControlState { DRIVE = 0, STOPPING, STOPPED, EMERGENCY };
```
and move it to `longitudinal_contrller_base`
https://github.com/autowarefoundation/autoware.universe/blob/main/control/trajectory_follower/include/trajectory_follower/pid_longitudinal_controller.hpp#L101-L111